src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -201,6 +198,12 @@ export const ArtistSeriesHeaderFragmentContainer = createFragmentContainer(
         artworksCountMessage
         descriptionFormatted(format: HTML)
         image {
+          _xs: cropped(height: 360, width: 360, version: "large") {
```

Comment:
Is there a convention for naming these versions based on breakpoint? We need one version at `xs` and one version for `sm` and greater (which is why I chose these).

Reply:
I think that's clear 👍